demos/http3/ossl-nghttp3-demo-server.c

```diff
@@ -68,6 +77,36 @@ static void add_id(uint64_t id, SSL *ssl, struct h3ssl *h3ssl)
     printf("Oops too many streams to add!!!\n");
     exit(1);
 }
+static void set_id_status(uint64_t id, int status, struct h3ssl *h3ssl)
+{
+    struct ssl_id *ssl_ids;
+    int i;
+
+    ssl_ids = h3ssl->ssl_ids;
+    for (i = 0; i < MAXSSL_IDS; i++) {
+        if (ssl_ids[i].id == id) {
+            printf("set_id_status: %llu to %d\n", (unsigned long long) ssl_ids[i].id, status);
+            ssl_ids[i].status = status;
+            return;
+        }
+    }
+    printf("Oops can't set status, can't find stream!!!\n");
+    exit(1);
```

Comment:
This should perhaps be an assert instead?  Seems like we don't want to just exit here

Reply:
like:
printf("Oops can't set status, can't find stream!!!\n");
assert(0);
?